app/models/post.rb

```diff
@@ -18,6 +18,24 @@ class Post < ActiveRecord::Base
 
   STATUS_CHOICES = %w(draft published archived retired)
 
+
+  include PgSearch
+  pg_search_scope :search, against: [:title, :body],
+    usig: {tsearch: {dictionary: "english"}}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside { missing.<br>Space inside } missing.